spec/support/boxes/line_item_box.cr

```diff
@@ -0,0 +1,8 @@
+require "uuid"
+
+class LineItemBox < BaseBox
+  def initialize
+    id  UUID.random.to_s
```

Comment:
I think this should be removed since ideally either LuckyRecord, or the database should generate it. I don't think the developer should have to generate it themselves